### ISSUE TITLE
prices from installation to installation

### ISSUE THREAD

**Issue by CloudscrapeR86** [xyzz/acquisition#212]
hello,

i have my whole folder saved from my previous installation

I have now installed acquisition on another machine.

How can I transfer, if possible, my whole pricing tags for every item in my stashes?

**Comment by xyzz** [comment]
copy contents of `%localappdata%\acquisition` to it

**Comment by CloudscrapeR86** [comment]
...that was NOT in the main folder, right? :/

**Comment by xyzz** [comment]
what's the main folder?

**Comment by CloudscrapeR86** [comment]
acquisition folder. and not appdata etc.
What if I insert the link to the original thread? can it READ from there?

**Comment by xyzz** [comment]
yes, it doesn't store any configs in "Program Files", it cannot read your data from the forums
